src/com/base2/ciinabox/PackerTemplateBuilder.groovy

```diff
@@ -78,9 +78,14 @@ class PackerTemplateBuilder implements Serializable {
     if (json) {
       chefProvisioner.json = json
     }
-
+    def use_cinc = false
```

Comment:
i would create a separate parameter to set packer to use cinc to make it more obvious to configure than prefixing the version with `cinc-`

Reply:
Created `useCinc` and `cincVersion` params for packer function